domains/_vercel.daim.json

```diff
@@ -0,0 +1,12 @@
+{
+  "owner": {
+    "username": "devdaim6",
+    "email": "daimdev6@gmail.com"
+  },
+  "records": {
+    "TXT": [
+      "vc-domain-verify=daim.is-a.dev,d88985a4876213470d49",
+      "vc-domain-verify=www.daim.is-a.dev,87167113d723d65b8332"
```

Comment:
This 2nd verification record should be under _vercel.www.daim.json, since that's what that domain is verify.